pgmpy/tests/test_inference/test_Sampling.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from pgmpy.models import MarkovModel
+from pgmpy.inference.Sampling import BayesianModelSampling
+from pgmpy.models import BayesianModel
+from pgmpy.factors import TabularCPD
+
+
+class TestBayesianModelSampling(unittest.TestCase):
+    def setUp(self):
+        self.bayesian_model = BayesianModel([('A', 'J'), ('R', 'J'), ('J', 'Q'),
+                                             ('J', 'L'), ('G', 'L')])
+        cpd_a = TabularCPD('A', 2, [[0.2], [0.8]])
+        cpd_r = TabularCPD('R', 2, [[0.4], [0.6]])
+        cpd_j = TabularCPD('J', 2,
+                           [[0.9, 0.6, 0.7, 0.1],
+                            [0.1, 0.4, 0.3, 0.9]],
+                           ['R', 'A'], [2, 2])
+        cpd_q = TabularCPD('Q', 2,
+                           [[0.9, 0.2],
+                            [0.1, 0.8]],
+                           ['J'], [2])
+        cpd_l = TabularCPD('L', 2,
+                           [[0.9, 0.45, 0.8, 0.1],
+                            [0.1, 0.55, 0.2, 0.9]],
+                           ['G', 'J'], [2, 2])
+        cpd_g = TabularCPD('G', 2, [[0.6], [0.4]])
+        self.bayesian_model.add_cpds(cpd_a, cpd_g, cpd_j, cpd_l, cpd_q, cpd_r)
+        self.sampling_inference = BayesianModelSampling(self.bayesian_model)
+        self.markov_model = MarkovModel()
+
+    def test_init(self):
+        with self.assertRaises(TypeError):
+            BayesianModelSampling(self.markov_model)
+
+    def test_forward_sample(self):
+        sample = self.sampling_inference.forward_sample(25)
+        self.assertEquals(len(sample), 25)
+        self.assertEquals(len(sample.columns), 6)
+        self.assertIn('A', sample.columns)
+        self.assertIn('J', sample.columns)
+        self.assertIn('R', sample.columns)
+        self.assertIn('Q', sample.columns)
+        self.assertIn('G', sample.columns)
+        self.assertIn('L', sample.columns)
+        self.assertEquals(set(sample.A), {'A_0', 'A_1', None})
```

Comment:
This line should read `self.assertTrue(set(sample.A).issubset({'A_0', 'A_1'}))`. With that change (and a similar one for `test_rejection_sample_basic`, all tests pass on my machine. 
However, the Travis-CI tests fail. From lines 573-74 of the Travis-CI log (Build #863), it seems that the contents of `sample.A` are `State(var='A', state=0)` and `State(var='A', state=1)`, instead of (the expected) `'A_0', 'A_1'`. Why could this be the case?